docs/articles_en/openvino_workflow/running_inference_with_openvino/stateful_models_intro.rst

```diff
@@ -0,0 +1,121 @@
+.. {#openvino_docs_OV_UG_stateful_models_intro}
+
+Stateful models and State API
+==============================
+
+.. toctree::
+    :maxdepth: 1
+    :hidden:
+
+    openvino_docs_OV_UG_ways_to_get_stateful_model
+
+What is Stateful Model?
+#######################
+
+Stateful model is a model which stores the specified data between and during model inferences in internal memory space usually called State or Variable.
+In contrast to usual **stateless** model, which purify all the calculated data after each inference call, **stateful**
+model preserve tensors saved in States.
+
+OpenVINO Stateful Model Representation
+######################################
+
+OpenVINO contains ReadValue/Assign operations to make a model Stateful.
+Each pair of ReadValue/Assign operates with State, known also as Variable,
+which is an internal memory buffer to store tensor data during and between model inference calls.
+ReadValue reads data tensor from State and returns it as output, Assign accepts data tensor as input and writes data to State
+to save data for the next inference call.
+
+These entities work as a coherent mechanism and help to achieve the next goals:
+
+1. Speed up execution of Model
+    Data in State is stored in the optimized form for OpenVINO plugins, which helps to execute model effectively.
+    But often requesting data from State might reduce the expected performance gains and even lead to losses,
+    so, it is expected that State mechanism will be used when data stored in State is not accessed frequently.
+
+2. Simplify user code
+    Typical scenarios as copying data from model's outputs to inputs in user code can be replaced with ReadValue/Assign operations.
+    OpenVINO will manage this copying internally.
+
+3. Specific scenarios
+    Several use cases require processing of data sequences. When length of a sequence is known and small enough,
+    we can process it with RNN like models that contain a cycle inside. But in some cases, like online speech recognition of time series
+    forecasting, length of data sequence is unknown. Then data can be divided in small portions and processed step-by-step. But dependency
+    between data portions should be addressed. For that, models save some data between inferences - state. When one dependent sequence is over,
+    state should be reset to initial value and new sequence can be started.
+
+OpenVINO has a special API to simplify work with Stateful models. State is automatically saved between inferences,
+and there is a way to reset state when needed. You can also read state or set it to some new value between inferences.
```

Comment:
Please rephrase it to be more fluent: we have already state that we have a dedicated API for that. And it looks to be a duplication. Provide more details (e.g. InferRequest class reference to identify where this API exists).